pkgs/servers/sql/postgresql/ext/pg_partman.nix

```diff
@@ -2,15 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "pg_partman";
-  version = "4.4.1";
+  version = "4.5.1";
 
   buildInputs = [ postgresql ];
 
   src = fetchFromGitHub {
     owner  = "pgpartman";
     repo   = pname;
     rev    = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev    = "v${version}";
```

Reply:
Why? This not being my package I'd rather mess with it as little as possible...